src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -148,6 +149,9 @@ protected FileMonitoringController(FilePath ws) throws IOException, InterruptedE
             if (status.exists()) {
                 try {
                     return Integer.parseInt(status.readToString().trim());
+                } catch (FileNotFoundException ignored) {
+                    /* File is locked. Ignore and just try again later. */
```

Comment:
this comment apears bogus - if the file is locked it will exist - so why is a `FileNotFoundException` thrown?